server/server.js

```diff
@@ -48,7 +48,7 @@ const bootstrapServer = async () => {
   // Not sure if this is needed, Chrome seems to grab the favicon just fine anyway
   // Maybe for cross-browser support
   app.use(logger("dev"));
-  app.use(favicon(path.resolve(env.PUBLIC_PATH, "favicon.ico")));
+  app.use(favicon(path.resolve(env.PUBLIC_PATH, 'images/icons', "favicon.ico")));
```

Comment:
https://nodejs.org/api/path.html#path_path_resolve_paths

ˋ'images', 'icons',ˋ Is the better solution as then the platform conform separator will be used.